### ISSUE TITLE
PartialReID project gets poor result

### ISSUE THREAD

**Issue by Thangbluee** [JDAI-CV/fast-reid#613]
Hi, many thanks to all the contributors for this project
I follow your config in PartialReid project and get poor result. 
The Rank-1 of all three datasets are around 66%, it is not the same as the your result. 
HELP!!!!!!!!!!!!

**Comment by shreejalt** [comment]
@Thangbluee  
I also encountered the same.
Did you solved it?
Default command uses 4 gpus to train. So I multiplied the learning rate by 4 and decreased the batch size by 4(from 256 to 64) then also I was unable to produce the results.

**Comment by Thangbluee** [comment]
@shreejalt 
I haven't solved it yet. 
Do you?
